chainerx/_docs/routines.py

```diff
@@ -1123,6 +1123,28 @@ def _docs_loss():
 .. seealso:: :func:`chainer.functions.gaussian_kl_divergence`
 """)
 
+    _docs.set_doc(
+        chainerx.sigmoid_cross_entropy,
+        """sigmoid_cross_entropy(x, t)
+
+Element-wise cross entropy loss for pre-sigmoid activations.
+
+Args:
+    x (~chainerx.ndarray): An array whose (i, j)-th element indicates the
+        unnormalized log probability of the j-th unit at the i-th example.
+    t (~chainerx.ndarray): An array whose (i, j)-th element indicates a signed
+        integer vector of ground truth labels 0 or 1. If ``t[i, j] == -1``,
+        corresponding ``x[i, j]`` is ignored. Loss is zero if all ground truth
+        labels are -1.
+
+Returns:
+    :class:`~chainerx.ndarray`: An array of the cross entropy.
+
+Note:
+    During backpropagation, this function propagates the gradient of the output
+    array to the input array ``x`` only.
```

Comment:
In this doc, could you make the arguments names consistent with that of the binding (i.e. `x1`, `x2`)? Otherwise, a function call would fail if passed as named arguments according to the docs i.e. (`x=..., t=...`).

Reply:
Fixed!